docs/modules/influxdb.md

```
@@ -0,0 +1,29 @@
+
+```bash
+npm install @testcontainers/influxdb --save-dev
+```
+
+## Examples
+
+The InfluxDB 2.x examples use the following libraries:
+
+- [@influxdata/influxdb-client](https://www.npmjs.com/package/@influxdata/influxdb-client)
+
+        npm install @influxdata/influxdb-client
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)